libraries/AP_Vehicle/ModeReason.h

```diff
@@ -70,4 +70,5 @@ enum class ModeReason : uint8_t {
   DEADRECKON_FAILSAFE = 50,
   MODE_TAKEOFF_FAILSAFE = 51,
   DDS_COMMAND = 52,
+  AUX_FUNCTION = 53,
```

Comment:
```suggestion
  AUX_FUNCTION = 1,
```
... and move it up next to the other one.

I think we can get away with making this an alias for `RC_COMMAND`.  And perhaps add a comment indicating what's going on...

Reply:
if its an alias we don't get the distinction in logs, which I think we do want

Reply:
Yeah, the goal was to get it in the log.

Reply:
> if its an alias we don't get the distinction in logs, which I think we do want

Ah, I'd missed the fact the mode switch continues to use RC_COMMAND.